packages/api-explorer/src/components/SchemaTabs/index.jsx

```diff
@@ -31,75 +39,108 @@ const styleItem = {
 
 function renderMissingSchema(nameSchema) {
   return (
-    <div style={{
-      padding: 10,
-      background: colors.schemaTabMissingSchemaBackground
-    }}
-    >
+    <div style={{padding: 10, background: colors.schemaTabMissingSchemaBackground}}>
       <FormattedMessage id={`schemaTabs.missing.${nameSchema.toLowerCase()}`} defaultMessage={`${nameSchema} schema not set`} />
     </div>
   )
 }
-export default class SchemaTabs extends Component {
+jsf.option({
+  failOnInvalidTypes: false,
+  useDefaultValue: true,
+})
 
+export default class SchemaTabs extends Component {
   constructor(props) {
     super(props)
     this.state = {
-      selected: 'request'
+      selected: 'example'
+    }
+  }
+
+  componentDidMount() {
+    const { operation, oas } = this.props
+    const schema = get(parametersToJsonSchema(operation, oas), '[0].schema', {})
+    refParser.dereference(resolveRootRef(schema), (err, schemaResolved) => {
+      if (!err) {
+        this.setState({schema: omit(['components'], schemaResolved)})
+        return
+      }
+      this.setState({schema: omit(['components'], schema)})
+    })
+  }
+
+  renderSchemaExample() {
+    try {
+      const {schema} = this.state
+      if (!schema) {
+        return renderMissingSchema('Example')
+      }
+
+      let example = get(schema, 'example')
+
+      if (!example) {
+        example = jsf.generate(schema)
+      }
+      return <JsonViewer missingMessage={'schemaTabs.missing.example'} schema={example} />
+    } catch (error) {
+      return <Alert type={'error'} message={error.message} />
     }
   }
 
   renderResponseSchema() {
     const { operation, oas } = this.props
-    return (
-      operation &&
-        operation.responses ? (
-          <ResponseSchema operation={operation} oas={oas} />
-        )
-        : renderMissingSchema('Response')
-    )
+    return operation && operation.responses ? (
+      <ResponseSchema operation={operation} oas={oas} />
+    ) : renderMissingSchema('Response')
   }
 
   renderRequestSchema() {
     const { operation, oas } = this.props
+    return operation && operation.requestBody ? (
+      <RequestSchema operation={operation} oas={oas} />
+    ) : renderMissingSchema('Request')
+  }
+
+  renderSchema () {
+    const {selected} = this.state
+    const selectedType = () => {
+      switch (selected) {
+        case 'request': {
+          return this.renderRequestSchema()
+        }
+        case 'response': {
+          return this.renderResponseSchema()
+        }
+        default: {
```

Comment:
case 'example', default could render null but should never be called

Reply:
k